disk/datadog_checks/disk/disk.py

```diff
@@ -105,6 +105,7 @@ def check(self, instance):
                 device_name = device_name.strip('\\').lower()
 
             tags.append('device:{}'.format(device_name))
+            tags.append('device_name:{}'.format(os.path.basename(device_name)))
```

Comment:
I think the `device_name` tag should be consistent and always be the `part.device` L90. But the tag `device` should be retrocompatible and follow the `use_mount` option.